### PR TITLE
Create 2021-05-14-scopri-come-contribuire-al-miglioramento-dei-serviz…

### PR DESCRIPTION
**Pull request by ruphy**
…i-pubblici-del-paese.md

## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create 2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 3a9981db71064f497b340ffca099dec99bf65667
</p>

File: _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Rilasciamo quindi oggi la prima versione di **uno strumento web **che aiuti gli aspiranti contributori a migliorare, con azioni concrete, il cuore del software libero per la pubblica amministrazione, bene comune di tutto il Paese.
+
+Le azioni proposte, in pieno spirito open source, possono essere realizzate già oggi da tutti coloro che desiderano supportare la transizione digitale della Pubblica Amministrazione in ottica di apertura, trasparenza, qualità ed efficienza.
+
+L’iniziativa è volta anche a risolvere una delle criticità emerse durante il recente [studio di ricerca](https://medium.com/designers-italia/il-valore-della-community-per-designers-italia-e-developers-italia-bd486dff2a6d) realizzato con la community. Gli utenti intervistati hanno infatti dichiarato che non sempre risulta semplice, per coloro che vogliono contribuire, comprendere quali azioni possano avere il maggiore impatto sulla vita dei cittadini, o come impiegare nel modo migliore le proprie competenze. Abbiamo così creato uno strumento facile e veloce, una sorta di __wizard__, che permette di trovare le attività più vicine alle proprie competenze ed inclinazioni.
```
Comment (bfabio) [inline _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md:22]:
```suggestion
L’iniziativa è volta anche a risolvere una delle criticità emerse durante il recente [studio di ricerca](https://medium.com/designers-italia/il-valore-della-community-per-designers-italia-e-developers-italia-bd486dff2a6d) realizzato con la community. Gli utenti intervistati hanno infatti dichiarato che non sempre risulta semplice, per coloro che vogliono contribuire, comprendere quali azioni possano avere il maggiore impatto sulla vita dei cittadini, o come impiegare nel modo migliore le proprie competenze. Abbiamo così creato uno strumento facile e veloce, una sorta di __wizard__, che permette di trovare le attività più vicine alle proprie competenze e inclinazioni.
```

File: _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: Scopri come contribuire al miglioramento dei servizi pubblici digitali del Paese
+subtitle: Una nuova iniziativa di Developers Italia e Designers Italia per la Settimana Aperta della PA
+date: 2021-05-14 10:00:00 +0100
+tags: ogp wizard designers community guida partecipazione
+categories: news
+author: Riccardo Iaconelli
+image: /assets/icons/logo-it.png
+locale: it
+---
+Chiamare a raccolta tutte le persone che a vario titolo e nelle modalità più diverse si impegnano o si vogliono impegnare a rendere la pubblica amministrazione italiana un luogo più aperto al confronto, più trasparente per i cittadini e più aperto all’innovazione. E’ questo l’obiettivo della **Settimana Aperta della Pubblica Amministrazione** (SAA), iniziativa promossa e coordinata dal Dipartimento della Funzione Pubblica nell’ambito della partecipazione italiana all’Open Gov Week.
```
Comment (bfabio) [inline _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md:12]:
```suggestion
Chiamare a raccolta tutte le persone che a vario titolo e nelle modalità più diverse si impegnano o si vogliono impegnare a rendere la pubblica amministrazione italiana un luogo più aperto al confronto, più trasparente per i cittadini e più aperto all’innovazione. È questo l’obiettivo della **Settimana Aperta della Pubblica Amministrazione** (SAA), iniziativa promossa e coordinata dal Dipartimento della Funzione Pubblica nell’ambito della partecipazione italiana all’Open Gov Week.
```

File: _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+L’evento consiste in cinque giornate (dal 17 al 21 maggio 2021) dedicate a sviluppare la cultura e la pratica della trasparenza, della partecipazione e dell’accountability sia nelle amministrazioni pubbliche che nella società.
+
+Riconoscendosi a pieno nello spirito che anima questa iniziativa collettiva, Developers Italia e Designers Italia intendono dare il proprio contributo facilitando coloro che desiderano **migliorare i servizi pubblici del Paese**.
+
+Rilasciamo quindi oggi la prima versione di **uno strumento web **che aiuti gli aspiranti contributori a migliorare, con azioni concrete, il cuore del software libero per la pubblica amministrazione, bene comune di tutto il Paese.
```
Comment (bfabio) [inline _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md:18]:
```suggestion
Rilasciamo quindi oggi la prima versione di **uno strumento web** che aiuti gli aspiranti contributori a migliorare, con azioni concrete, il software libero per la pubblica amministrazione, bene comune di tutto il Paese.
```

File: _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+date: 2021-05-14 10:00:00 +0100
+tags: ogp wizard designers community guida partecipazione
+categories: news
+author: Riccardo Iaconelli
+image: /assets/icons/logo-it.png
+locale: it
+---
+Chiamare a raccolta tutte le persone che a vario titolo e nelle modalità più diverse si impegnano o si vogliono impegnare a rendere la pubblica amministrazione italiana un luogo più aperto al confronto, più trasparente per i cittadini e più aperto all’innovazione. E’ questo l’obiettivo della **Settimana Aperta della Pubblica Amministrazione** (SAA), iniziativa promossa e coordinata dal Dipartimento della Funzione Pubblica nell’ambito della partecipazione italiana all’Open Gov Week.
+
+L’evento consiste in cinque giornate (dal 17 al 21 maggio 2021) dedicate a sviluppare la cultura e la pratica della trasparenza, della partecipazione e dell’accountability sia nelle amministrazioni pubbliche che nella società.
```
Comment (bfabio) [inline _posts/it/2021-05-14-scopri-come-contribuire-al-miglioramento-dei-servizi-pubblici-del-paese.md:14]:
```suggestion
L’evento consiste in cinque giornate (dal 17 al 21 maggio 2021) dedicate a sviluppare la cultura e la pratica della trasparenza, della partecipazione e della responsabilità sia nelle amministrazioni pubbliche che nella società.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**